grunt-ngdocs-index.tmpl

```diff
@@ -31,7 +31,7 @@
       }
 
       addTag('base', {href: baseUrl});
-      addTag('link', {rel: 'stylesheet', href: 'css/prettify.css', type: 'text/css'});
+      // addTag('link', {rel: 'stylesheet', href: 'css/prettify.css', type: 'text/css'});
```

Comment:
In that case I think we should add a comment above as to why it is commented out and still here.